src/legacy/state/lists/updater.ts

```diff
@@ -35,7 +35,12 @@ export default function Updater(): null {
   }, [fetchList, isWindowVisible, lists])
 
   // fetch all lists every 10 minutes, but only after we initialize provider
-  useInterval(fetchAllListsCallback, provider ? 1000 * 60 * 10 : null)
+  useInterval({
+    callback: fetchAllListsCallback,
+    name: 'ListsUpdater',
+    delay: provider ? 1000 * 5 : null, // provider ? ms`10m` : null,
```

Comment:
This is changing the poll frequency.
Is it intentional to go from 10min to 5 seconds?